lang/c/c.talon

```diff
@@ -7,7 +8,7 @@ tag(): user.code_comment_line
 tag(): user.code_comment_block_c_like
 tag(): user.code_data_bool
 tag(): user.code_data_null
-tag(): user.code_functions
+# tag(): user.code_functions
```

Comment:
Could you explain why you have disabled this?

Reply:
That is a good question. I remember being upset that `funky` didn't have any means to sneak in a return type for the function you're defining. Started experimenting with a more specific `funky` command, but from the looks of it I never finished that...
```
[<user.c_qualifier_list>] <user.c_type> funky <user.text>:
    qualifiers = c_qualifier_list or ""
    insert("{qualifiers} {text}")
```
This has been alleviated for me by defining the function definition snippet to use the `auto` return type and simply using `snip funky` for everything. Maybe I should just reinstate that tab.